### PR TITLE
Tests for the routing connector

### PR DESCRIPTION
**Pull request by phliar**

### CODE REVIEW

File: connectors/routing/config_test.go [hunk 1]
```diff
@@ -27,42 +27,41 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-// TestBasicConfig test the basic yaml file conversion
-func TestBasicConfig(t *testing.T) {
-	yamlFile := `
+var yamlFile = `
 routers:
-# routers structure looks like:
```
Comment (awaniim) [inline connectors/routing/config_test.go]: why remove these comments?

**Comment by skywhat** [comment]
> The spec for the routing connector:
> 
> ```
> // Config for the routing connector is a "case statement" of scope names, and each entry is a list
> // of assigments "pattern" -> engine-name.
> //
> // Example:
> //
> // routers:
> // - production:
> //     serviceA: cassandra
> //     serviceX: schemaless
> //     *: dosa
> // - development:
> //     *: dosa_dev
> //     serviceB: cassandra
> //     serviceX: schemaless
> // - ebook:
> //     '*': ebook
> //     apple.*: ebook
> //     ebook-store: ebook
> // - default:
> //     sless_*: schemaless
> //     "*": dosa_dev
> //
> // A pattern is not a regular expression: only prefixes may be specified (i.e. trailing "*").
> // Literal strings (no "*") sort before a pattern.
> // "default" is an alias for "*" and sorts after everything else.
> ```

where do u get the routing grammar, do we need to add the comment in this `routing` file?

**Comment by phliar** [comment]
BTW this is for information only, no need to accept.....

**Comment by phliar** [comment]
Not for commit

**Comment by phliar** [comment]
> where do u get the routing grammar, do we need to add the comment in this `routing` file?

It is not documented anywhere, I figured this out by reading the code. (Obviously the final diff will include the spec that we are trying to parse.)

**Pull request closed without merging**